### ISSUE TITLE
[reactpoll] Request: Poll Results show voters

### ISSUE THREAD

**Issue by BMCSamael** [flapjax/FlapJack-Cogs#58]
The tally is great but since it removes the emoji at the end of the vote (as it should) its impossible to see who voted which way.

It would be great to be able to display that so its 
1- 2 votes - ['Voter1','Voter2']
2- 0 Votes - ['']

**Comment by skeith** [comment]
if you need a poll that show who vote for an option, you better use tmerc's survey cog

**Comment by flapjax** [comment]
Yes, do try [tmerc's survey](http://cogs.red/cogs/tmercswims/tmerc-cogs/survey/) and let us know if that suits your needs. I would rather not make changes to this cog. It was just intended to be a simple fork of the general module's polls, to use reactions instead. Red v3 will likely have reaction polls built in and I intend to deprecate this cog at that time.
